dev_guide/builds/build_environment.adoc

```diff
@@ -16,18 +16,24 @@ toc::[]
 == Overview
 
 As with pod environment variables, build environment variables can be defined in
-terms of references to other resources/variables using the xref:../../dev_guide/downward_api.adoc#dev-guide-downward-api[Downward API].
-However, there are some exceptions as noted below.
+terms of references to other resources and variables using the
+xref:../../dev_guide/downward_api.adoc#dev-guide-downward-api[Downward API].
+However, there are some exceptions as noted in this section.
 
 [NOTE]
 ====
 You can also manage environment variables defined in the `BuildConfig` with the
-xref:../../dev_guide/environment_variables.adoc#dev-guide-environment-variables[`oc set env`] command.
+xref:../../dev_guide/environment_variables.adoc#dev-guide-environment-variables[`oc
+set env`] command.
 ====
 
 [[using-build-fields-as-environment-variables]]
 == Using Build Fields as Environment Variables
 
+You can inject information about the build object by setting the `fieldPath`
+environment variable source to the `jsonpath` of the field that you are
+interested in obtaining the value.
```

Comment:
i know we supposedly aren't supposed to end sentences with prepositions (oddly this was a topic at lunch today too) but i really think an "from" at the end of this sentence would make it more readable.

or "of the field from which you are interested in obtaining the value" if you want to avoid the tail preposition.